linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/oval/shared.xml

```diff
@@ -0,0 +1,49 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Netrc User File In The Home Directory Must Not be group or world
+    accessible") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_netrc_file_permissions"
+                 comment="Netrc User File In The Home Directory Must Not be group or world
+                          accessible"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_netrc_file_permissions_objects" version="1">
+    <unix:username datatype="string" operation="pattern match">.*</unix:username>
+    <filter action="include">state_{{{ rule_id }}}_interactive_gids</filter>
```

Comment:
Please, ensure the `id` in the state declaration is equivalent. This is causing the legit errors in CI tests.
Also, maybe it would be better to change `gids` by `uids` in `..._interactive_gids`.

Reply:
Should be unified in [5ef9feb](https://github.com/ComplianceAsCode/content/pull/9988/commits/5ef9feb3aeae2d369ea94d2b6ffca574d2010a78)